File: project3/Team3/projectBQ/src/main/java/bitc/full502/projectbq/dto/ItemDto.java
package bitc.full502.projectbq.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ItemDto {

    //    비품 List Dto (20250917 완료)
    private Long id;

    private Long categoryId;

    private String categoryName;

    private String code = "";

    private String name;

    private String manufacturer;

    private Long price;

    private Long allQuantity = 0L;

    private LocalDateTime addDate;

    private String addUser;

    private String approveUser = "";

    private String addUserName;

    private String approveUserName;

    private String status;
}

File: project3/Team3/App_BQ/app/src/main/java/bitc/full502/app_bq/data/model/ItemDto.kt
package bitc.full502.app_bq.data.model


data class ItemDto(
    val id: Long,
    val categoryId: Long,
    val categoryName: String,
    val code: String = "",
    val name: String,
    val manufacturer: String,
    val price: Long,
    val addDate: String?,  // 앱에서는 String으로 받는게 편합니다
    val addUser: String,
    val approveUser: String = "",
    val status: String?
)